webhook_to_fedora_messaging/config/config.toml

```diff
@@ -0,0 +1,60 @@
+[flaskapp]
```

Comment:
Is this file an example config file? If so, you can put it in the root directory with a `.example` suffix, it's more discoverable that deep inside the source code.